docs/configurations.md

```diff
@@ -76,4 +76,7 @@ summary: "Here we list all possible configurations and what they mean"
         - [usePrefix](#usePrefix) () <br/>
         <span style="color:grey">Standard prefix for all metrics</span>
 
+    - [S3Configs](s3_hoodie.html) (Hoodie S3 Configs) <br/>
```

Comment:
I have added the permalink as -
`permalink: s3_hoodie.html`

I have tested locally on Jekyll 👍